examples/DgraphExample/ExecuteDQLQuery.cs

```diff
@@ -5,7 +5,7 @@ namespace DgraphExample
 {
     class ExecuteDQL
     {
-        public static async Task<string> Query(DgraphClient Client, string query)
+        public static async Task<string> Query(IDgraphClient Client, string query)
```

Comment:
Does this change the API? what is the difference between DgraphClient & IDgraphClient?

Reply:
IDgraphClient is an interface, DgraphClient is the implementation. Users will interact with the interface instead of the class directly. It hides any implementation details and lets them mock the client for testing if they want.